src/Admin/ArticleAdmin.php

```diff
@@ -91,7 +91,6 @@ protected function configureFormFields(FormMapper $formMapper)
                 ])
                 ->add('slug', null, [
                     'label' => 'URL de publication',
-                    'disabled' => !$slugEditable,
                     'help' => $slugEditable ? 'Ne spécifier que la fin : http://en-marche.fr/articles/[votre-valeur]<br />Doit être unique' : 'Non modifiable car publié',
```

Comment:
This help message should always be the same now that the field is never disabled